drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -82,8 +87,8 @@ def active
     operating_end_date >= Date.current
   end
 
-  def enrollments
-    Hmis::Hud::Enrollment.in_project_including_wip(id, project_id)
+  def enrollments_including_wip
+    Hmis::Hud::Enrollment.where(data_source_id: data_source_id).in_project_including_wip(id, project_id)
```

Comment:
This is probably good, but you could also write it like
```suggestion
    enrollment_ids = enrollments.pluck(:id)
    wip_ids = wip_enrollments.pluck(:id)
    Hmis::Hud::Enrollment.where(id: enrollment_ids + wip_ids)
```

Not sure that's better though, just moves the logic into the project, which maybe isn't good..  Obviously, I'm on the fence about that.

Reply:
I think it makes more sense to have the logic in the project, and it's way easier to read than the `in_project_including_wip` scope. I removed the `in_project_including_wip` scope entirely with https://github.com/greenriver/hmis-warehouse/pull/3037/commits/59b329cce55560f6808c227307d398595572319a